### PR TITLE
Feature: SCSS Mixins added

### LINKED ISSUE #17: SCSS mixins creation
This issue is consists in create all the necessary Mixins that will help to keep the consistency in the layout of the project in both modes (dark & clear) with te help of the css variables.

### PR DESCRIPTION
**Pull request by andresV74**
SCSS Mixins added

I added the most significant mixins in the file **_mixins.scss**.

Please check the file and tell me if there are something missing

### CODE REVIEW

**Review by georgehossa — APPROVED** [review]

File: src/styles/_colors.scss [hunk 1]
```diff
@@ -1,11 +1,13 @@
 html {
-  --base-color-background: #282c34;
-  --base-color-link: #61dafb;
-  --base-color: white;
+	--base-color-background: #282c34;
+	--base-color-link: #61dafb;
+	--base-color: white;
+	--base-color-button: #929292;
```
Comment (georgehossa) [inline src/styles/_colors.scss]: I think these custom properties must be declared in `:root` to be available in all document.
And if you can please define all colors variables from the design and these variables can be used in these lines.
I attach the list of all colors in the design:
095BE094-EFE7-4E00-B344-FD168D747E1E

let me know if not clear and if you have any doubt.

**Review by georgehossa — COMMENTED** [review]
I make some suggestions that I think can improve the experience with SASS for the team.

File: src/styles/_colors.scss (hunk 1 above)
Comment (wolftrax5) [reply src/styles/_colors.scss]: it's a good suggestion,  but 1 question at do we have the negative/contrast of these colors I mean if @andresV74  put
`--background-card: white,` on the for the dark mode on the tag `html[data-theme="dark"] ` should be --background-card: black, if the color didn't change keep it

**Review by wolftrax5 — CHANGES_REQUESTED** [review]
pls check the point of the colors, because with the mixing will be the base of the project

**Review by wolftrax5 — APPROVED** [review]

File: src/styles/_colors.scss (hunk 1 above)
Comment (andresV74) [reply src/styles/_colors.scss]: I already did the changes and suggestions that you made and push those changes to gitHub, but I'm not quite sure if I have to make a new PR with those changes. Please let me know

File: src/styles/_colors.scss [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 	--base-color-button: #929292;
 }
 
-html[data-theme="dark"] {
-	--base-color-background: white;
+:root[data-theme="dark"] {
+	--base-color-background: #282c34;
 	--base-color-link: #fe016c;
 	--base-color: #232323;
 	--base-color-button: #273239;
```
Comment (georgehossa) [inline src/styles/_colors.scss]: I think we got a misunderstood when I say the variables must be in the `:root`  What I want to say we have to declare all colors in the `:root` and later use them on the `html` element.
Some thing like this:
```css
/* Only CSS Custom Properties */
:root {
  --blue: #0096ca;
  --darkBg: #222;
  --lightBg: #fff;
}

html {
 --base-color-background: var(--lightBg);
} 

html[data-theme="dark"] {
 --base-color-background: var(--darkBg);
} 
```
```scss
/* CSS Custom Properties and SASS */
$blue: #0096ca;
$darkBg: #222;
$lightBg: #fff;

html {
 --base-color-background: #{$lightBg};
} 

html[data-theme="dark"] {
 --base-color-background: #{$darkBg};
} 
```

**Review by georgehossa — CHANGES_REQUESTED** [review]
Look I made some suggestion

**Review by wolftrax5 — APPROVED** [review]

**Review by wolftrax5 — APPROVED** [review]

**Review by georgehossa — APPROVED** [review]
I think is done.

**Comment by andresV74** [comment]
I restored this branch because I was reading the Sass documentation and fin out that what we needed wasn't Mixins but Extends so I fixed that file but kept its name as `_mixins.scss`.

Here's a list of the extends:

- `%shadow-banners`
- `%shadow-cards` (for the cards inside the banners)
- `%banners`
- `%title` (for the ones with info icon)
- `%button` (round buttons)
- `%symptomsCard`

I also added the light version colors and adjusted the html variables in the `_colors.scss file`.

**Pull request merged**